[IMP] Inventory: Rewrite aging report as stock valuation report (#11720)
odoo/documentation: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/aging.rst

=========================
Stock valuation dashboard
=========================

.. |SVLs| replace:: :abbr:`SVLs (stock valuation layers)`

> When a company has physical assets, such as inventory, they often want to know approximately how

Author: Lovely suggestion, and thanks for the reminder of 1. Definition 2. Use-case 3. How-to structure...super helpful! I ended up with a sort of hybrid approach.

:doc:`Stock valuation <../../product_management/inventory_valuation/using_inventory_valuation>`
typically utilizes one of two accounting systems:

- **Perpetual**: The inventory is constantly (perpetually) being updated, and the value is
  constantly changing.
- **Periodic**: The inventory value is checked on an occasional (periodic) basis, and the value is
  set at this occasional time.

Using :ref:`tracked inventory <inventory/product_management/tracking-inventory>` in Odoo
necessitates a *perpetual* inventory accounting system because of the need to know when and where
inventory exists, and how much of it is available or forecasted. There are a few common :ref:`stock
valuation methods <inventory/warehouses_storage/costing_methods>` used in Odoo: standard price,
average cost (AVCO), and first in, first out (FIFO) accounting. It is important to know that the
valuation method chosen for a product impacts the calculation of several fields in the stock
valuation reports.

Open the dashboard
==================

Odoo's Stock Valuation dashboard displays the financial value of all tracked inventory, according

> to each product's stock valuation method. This report can provide insights into sunken purchase

Author: Love this- thank you! Implemented.

costs or delays in profitability. To access the dashboard, go to :menuselection:`Inventory app -->
Reporting --> Valuation`.

.. important::
   The *Reporting* menu in *Inventory* is only accessible to users with :doc:`admin access
   <../../../../general/users/access_rights>`.

This dashboard has three different views, or inventory reports — :ref:`list view (i.e. the default,
stock valuation report) <inventory/warehouses_storage/valuation-report>`, :ref:`pivot view (i.e. the
stock aging report) <inventory/warehouses_storage/aging-report>`, and :ref:`graph view
<inventory/warehouses_storage/graph-view>`. Each view can be customized with different fields to
break down inventory valuation by product, operation type, date, or company.

All three views can be filtered by various fields. To apply filters, click into the search bar at
the top of the report, or click the drop-down arrow next to it. For example, selecting the filter
:guilabel:`Has Remaining Qty` will show only products that are currently in stock.

.. _inventory/warehouses_storage/valuation-report:

> List view: stock valuation

Author: Talked with Felicia - I'm going to leave this one lowercase to parallel the 'Pivot view: stock aging' header below. For that one, I don't think I can capitalize 'stock aging' because the page is not actually titled 'Stock Aging'. But I see what you're saying and had thought about that for a minute as well!

==========================

By default, the :guilabel:`Stock Valuation` dashboard displays in *list view*, represented by the
:icon:`oi-view-list` :guilabel:`(list)` icon. This view is essentially a *Stock Valuation Report*,
which shows a detailed record of stock movements and their valuations.

Configure
---------

The following columns are displayed by default:

- :guilabel:`Date`: the date and time when the stock move was created. The valuation report is

>   sorted by this field, emphasizing the importance of time when valuing inventory.

Author: Yes, agreed! Implemented.

- :guilabel:`Reference`: the reference document associated with this stock move (e.g., a warehouse
  receipt, a delivery order, or a manual inventory adjustment).
- :guilabel:`Product`: the product that is being moved and valued.
- :guilabel:`Quantity`: the number of units by which this product's stock has increased or
  decreased in this particular stock move.
- :guilabel:`Total Value`: the value of the product's stock in this particular stock move,
  calculated by multiplying the :guilabel:`Quantity` and :guilabel:`Unit Value`.

.. note::
   If a :guilabel:`Reference` document includes several goods, there will be a separate line item
   generated on the Stock Valuation Report for each good.

> There are additional fields that can be added to this view to provide more insight into the stock's

Author: Implemented!

their stock valuation. Specifically, the stock moves that generate SVLs are warehouse receipts,
deliveries, dropshipping orders, and dropshipping returns. These stock moves must first be validated
(by clicking the :guilabel:`Validate` button) for the SVL to be created.

If a product's inventory valuation method changes on the product form, new line items are generated
on the Stock Valuation Report to reflect the resulting SVLs. For example, if the valuation method
changes from *standard price* to either *AVCO* or *FIFO* accounting, *revaluation entries* will be
automatically posted to reflect the change in pricing for goods that remain in stock. One entry will
be negative to "remove" the old pricing, and the second entry will be positive to record the new
pricing. These entries are connected to journal entries in Odoo's **Accounting** app.

Below is an example of what the Stock Valuation Report table shows when a few stock moves have
occurred for a product using standard price accounting.

.. image:: aging/before-val-method-change.png
   :alt: Remaining value and quantity fields are calculated based on SVLs.

> Whereas the following image depicts what the Stock Valuation Report table might look like after a

Author: I switched "whereas" for "conversely"! Trying to highlight the comparison to the screenshot before.

   to `90.00`, once the second stock move is recorded. This change reflects that, although 100
   sweaters were originally purchased, only 90 of those sweaters remain in stock and should be
   counted in the valuation. Similarly, the :guilabel:`Remaining Value` will drop from `$500.00` to
   `$450.00`. The :guilabel:`Total Value` will remain at `$500.00`, regardless of subsequent
   transactions.

   On the other hand, the :guilabel:`Remaining Quantity` of the second stock move line item will be
   recorded and remain at `0.00` because the quantity of `-10.00` was sold. In the system, because
   the SVL was a sale, there is no stock left that needs to be valued from that transaction.

   .. image:: aging/remaining-val-quant.png
      :alt: Remaining value and quantity fields are calculated based on SVLs.

Change the valuation date
-------------------------

The :guilabel:`Valuation At Date` button, located in the top-left corner of the Stock Valuation
page, reveals a pop-up window. In this pop-up, select a date to act as a "cut-off" for the stock
moves to be valued.

.. note::
   The value of the stock moves will not be point-in-time for any dates chosen in the past. In other

>    words, the stock move shown when selecting a past date will still display the current on-hand

Reviewer: This is purely stylistic so optional suggestion, but I don't think this needs qualifying language since the preceding sentence flows logically into this one

   value of its products.

.. _inventory/warehouses_storage/aging-report:

> Pivot view: stock aging

Reviewer: ```suggestion
Pivot view: Stock Aging
```

=======================

From the Stock Valuation dashboard, access pivot view by clicking the :icon:`oi-view-pivot`
:guilabel:`(pivot table)` icon. This view is essentially a *Stock Aging Report*, and it shows the
on-hand quantity and value of inventory by purchase date, which can help monitor products with
expiration dates.

Configure
---------

By default, the pivot view shows the value of all *product categories* by *day and month*. Clicking
the :icon:`fa-plus-square` :guilabel:`(plus)` icon in each column or row will reveal a drop-down

> list of options to further tabulate the table and create a more granular breakdown of the inventory

Reviewer: ```suggestion
list of options to further populate the table and create a more granular breakdown of the inventory
```

valuation. The drop-down options include: :guilabel:`Product`, :guilabel:`Lot/Serial Number`,
:guilabel:`Product Category`, :guilabel:`Date`, :guilabel:`Company`, or :guilabel:`Add Custom Group`
Clicking the :icon:`fa-minus-square-o` :guilabel:`(minus)` icon collapses the field back to an empty
state.

In the table, the :guilabel:`Remaining Qty` column displays the number of on-hand items, and
:guilabel:`Remaining Value` displays the total cost of purchasing these items.

.. image:: aging/stock-aging-report.png
   :alt: Stock aging report, showing product row items and day columns.

.. _inventory/warehouses_storage/graph-view:

Graph view
==========

The stock value can be depicted graphically by clicking the :icon:`fa-area-chart` :guilabel:

> `(graph)` icon. By default, the graph is displayed in :icon:`fa-line-chart` line chart view and

Reviewer: You already noted this but it didn't show up in the build, maybe the linebreak?

- :guilabel:`Quantity`: the number of units by which this product's stock has increased or
  decreased in this particular stock move.
- :guilabel:`Total Value`: the value of the product's stock in this particular stock move,
  calculated by multiplying the :guilabel:`Quantity` and :guilabel:`Unit Value`.

.. note::
   If a :guilabel:`Reference` document includes several goods, there will be a separate line item
   generated on the Stock Valuation Report for each good.

There are additional fields that can be added to this view to provide more insight into the stock's
valuation. To add fields, click the :icon:`oi-settings-adjust` :guilabel:`(Adjust Settings)` icon,
and select the desired fields:

- :guilabel:`Lot/Serial Number`: the uniquely identifying lot or serial number for this product.
- :guilabel:`Company`: for businesses that operate with multiple companies, this field displays the
  company by which this stock move took place.
- :guilabel:`Remaining Quantity`: the number of units remaining for this valuation of the product,
  after demand has been accounted for (even from other stock moves). This field can be especially

>   helpful for FIFO and AVCO accounting, as it conveys which units of stock came into a warehouse

Reviewer 2: Excluding the first time you defined AVCO and FIFO
![image](https://github.com/user-attachments/assets/2d0f94c3-7e56-4055-9eaf-861bd7628f09)

Make sure to use the [abbreviation tags](https://www.odoo.com/documentation/18.0/contributing/documentation/rst_guidelines.html#abbreviations) every time you refer to them as concepts (: (guilabels when we're talking about fields)

Reviewer 2: ```suggestion
  helpful for :abbr:`FIFO (First In, First Out)` and :abbr:`AVCO (Average Cost)` accounting, as it conveys which units of stock came into a warehouse
```

  first and the value of said stock.

> - :guilabel:`Unit Value`: the cost of one unit of the product (**not** the price to consumers).

Reviewer 2: Maybe this helps clarify it a little more?
```suggestion
- :guilabel:`Unit Value`: the cost of one unit of the product for the company (**not** the price to consumers).
```

- :guilabel:`Description`: a description of the reason for this stock valuation (typically, a stock

>   move has occurred). By default, this field is set as the concatenation of the :guilabel:
>   `Reference` and :guilabel:`Product` fields. However, the field may also display other important

Reviewer 2: Extra space
```suggestion
  move has occurred). By default, this field is set as the concatenation of the :guilabel:`Reference` and :guilabel:`Product` fields. However, the field may also display other important
```

  messages for this line item, such as a note stating that the line item is an adjustment due to a
  change in the product's inventory valuation method.
- :guilabel:`Remaining Value`: the value of this product's current stock levels for this particular
  stock move, after demand has been accounted for. Along with :guilabel:`Remaining Quantity`, this
  field can be especially helpful for FIFO and AVCO accounting, as they convey which stock came
  into a warehouse first and the value of said stock.

.. image:: aging/stock-valuation-report.png
   :alt: Stock valuation report.

> 

Reviewer 2: If you accepted the suggestion to link to this section in an earlier comment (:
```suggestion
.. _inventory/warehouses_storage/svl:
```

Stock valuation layers (SVLs)
-----------------------------

Each line item in the :guilabel:`Stock Valuation` report represents a record in Odoo's system known

> as a *stock valuation layer (SVL)*. SVLs are generated when products move in a way that impacts

Reviewer 2: I'll change this one, but remember to add `abbr` tags to all instances of "SVL"!
```suggestion
as a *stock valuation layer (SVL)*. :abbr:`SVLs (stock move layers)` are generated when products move in a way that impacts
```

their stock valuation. Specifically, the stock moves that generate SVLs are warehouse receipts,
deliveries, dropshipping orders, and dropshipping returns. These stock moves must first be validated
(by clicking the :guilabel:`Validate` button) for the SVL to be created.

If a product's inventory valuation method changes on the product form, new line items are generated

> on the Stock Valuation Report to reflect the resulting SVLs. For example, if the valuation method

Reviewer 2: ```suggestion
on the *Stock Valuation Report* to reflect the resulting SVLs. For example, if the valuation method
```

   sweaters were originally purchased, only 90 of those sweaters remain in stock and should be
   counted in the valuation. Similarly, the :guilabel:`Remaining Value` will drop from `$500.00` to
   `$450.00`. The :guilabel:`Total Value` will remain at `$500.00`, regardless of subsequent
   transactions.

   On the other hand, the :guilabel:`Remaining Quantity` of the second stock move line item will be
   recorded and remain at `0.00` because the quantity of `-10.00` was sold. In the system, because
   the SVL was a sale, there is no stock left that needs to be valued from that transaction.

   .. image:: aging/remaining-val-quant.png
      :alt: Remaining value and quantity fields are calculated based on SVLs.

Change the valuation date
-------------------------

To specify a "cut-off" date and time for the stock moves to be valued, click the
:guilabel:`Valuation At Date` button, located in the top-left corner of the Stock Valuation page.

.. note::
   The value of the stock moves will not be point-in-time for any dates chosen in the past. The
   stock move shown when selecting a past date will still display the current on-hand value of its

>    products.

Reviewer 2: The "Valuation At Date" button is designed to help businesses know exactly *what* they had on-hand at a specific date and the valuation of those items.  Your explanation of a "cut-off" date is helpful, but it could be clarified a bit further.

Since our documentation is Google-translated into other languages, I wonder if "cut-off" might not translate well. It might be better to rephrase it using more universally understood terms with straightforward dictionary definitions. Could you try reworking this sentence?

.. _inventory/warehouses_storage/aging-report:

Pivot view: stock aging
=======================

> From the Stock Valuation dashboard, access pivot view by clicking the :icon:`oi-view-pivot`

Reviewer 2: ```suggestion
From the *Stock Valuation* dashboard, access pivot view by clicking the :icon:`oi-view-pivot`
```

> :guilabel:`(pivot table)` icon. This view is essentially a *Stock Aging Report*, and it shows the

Reviewer 2: ```suggestion
:guilabel:`(pivot)` icon. This view is essentially a *Stock Aging Report*, and it shows the
```

  constantly changing.
- **Periodic**: The inventory value is checked on an occasional (periodic) basis, and the value is
  set at this occasional time.

Using :ref:`tracked inventory <inventory/product_management/tracking-inventory>` in Odoo
necessitates a *perpetual* inventory accounting system because of the need to know when and where
inventory exists, and how much of it is available or forecasted. There are a few common :ref:`stock
valuation methods <inventory/warehouses_storage/costing_methods>` used in Odoo: *standard price*,
*average cost* (AVCO), and *first in, first out* (FIFO) accounting. It is important to know that the
valuation method chosen for a product impacts the calculation of several fields in the stock
valuation reports.

Open the dashboard
==================

Odoo's *Stock Valuation* dashboard displays the financial value of all tracked inventory, according
to each product's stock valuation method. This report can provide insights into potential issues in
the supply chain, such as sunken purchase costs or delays in profitability. To access the dashboard,
go to :menuselection:`Inventory app --> Reporting --> Valuation`.

.. important::

>    The *Reporting* menu in *Inventory* is only accessible to users with :doc:`admin access
>    <../../../../general/users/access_rights>`.

Reviewer 3: Since we are already updating this doc... please update the format here :)
```suggestion
   The :menuselection:`Reporting` menu in **Inventory** is only accessible to users with :doc:`admin
   access <../../../../general/users/access_rights>`.
```

   generated on the report for each good.

There are additional fields that can be added to this view to provide more insight into the stock's
valuation. To add fields, click the :icon:`oi-settings-adjust` :guilabel:`(adjust)` icon, and select
the desired fields:

- :guilabel:`Lot/Serial Number`: the uniquely identifying lot or serial number for this product.
- :guilabel:`Company`: for businesses that operate with multiple companies, this field displays the
  company by which this stock move took place.
- :guilabel:`Remaining Quantity`: the number of units remaining for this valuation of the product,
  after demand has been accounted for (even from other stock moves). This field can be especially
  helpful for :abbr:`FIFO (First In, First Out)` and :abbr:`AVCO (Average Cost)` accounting, as it
  conveys which units of stock came into a warehouse first and the value of said stock.
- :guilabel:`Unit Value`: the cost of one unit of the product for the company (**not** the price to
  consumers).
- :guilabel:`Description`: a description of the reason for this stock valuation (typically, a stock

>   move has occurred). By default, this field is set as the concatenation of the :guilabel:
>   `Reference` and :guilabel:`Product` fields. However, the field may also display other important
>   messages for this line item, such as a note stating that the line item is an adjustment due to a
>   change in the product's inventory valuation method.

Reviewer 3: Please make sure to break lines in RST tags only where there is a space!
For example:
```
...the :guilabel:❌`Reference` and...
...the ✅:guilabel:`Reference` and...
``` 
or another example of where you can break a line:
```
:menuselection:❌`Inventory app --> Reporting --> Valuation`
:menuselection:`Inventory✅app --> Reporting --> Valuation`
```

```suggestion
  move has occurred). By default, this field is set as the concatenation of the
  :guilabel:`Reference` and :guilabel:`Product` fields. However, the field may also display other
  important messages for this line item, such as a note stating that the line item is an adjustment
  due to a change in the product's inventory valuation method.
```

  :abbr:`AVCO (Average Cost)` accounting, as they convey which stock came into a warehouse first and
  the value of said stock.

.. tip::
   Some of these settings may not appear unless first enabled in Odoo's **:doc:`Settings
   <../../../general>`** application.

.. image:: aging/stock-valuation-report.png
   :alt: Stock valuation report.

.. _inventory/warehouses_storage/svl:

Stock valuation layers (SVLs)
-----------------------------

Each line item in the :guilabel:`Stock Valuation` report represents a record in Odoo's system known
as a *stock valuation layer (SVL)*. :abbr:`SVLs (stock move layers)` are generated when products
move in a way that impacts their stock valuation. Specifically, the stock moves that generate
:abbr:`SVLs (stock move layers)` are warehouse receipts, deliveries, dropshipping orders, and
dropshipping returns. These stock moves must first be validated (by clicking the
:guilabel:`Validate` button) for the :abbr:`SVL (stock move layer)` to be created.

If a product's inventory valuation method changes on the product form, new line items are generated
on the :guilabel:`Stock Valuation` report to reflect the resulting :abbr:`SVLs (stock move layers)`.

> For example, if the valuation method changes from *standard price* to either *:abbr:`AVCO (Average
> Cost)`* or *:abbr:`FIFO (First In, First Out)`* accounting, *revaluation entries* will be
> automatically posted to reflect the change in pricing for goods that remain in stock. One entry will
> be negative to "remove" the old pricing, and the second entry will be positive to record the new
> pricing. These entries are connected to journal entries in Odoo's **Accounting** app.

Reviewer 3: Syntax error: you can't stack formatting (italics, bold, etc) on RST roles (abbr, guilabel, menuselection, doc, etc). Please avoid this :)
```suggestion
For example, if the valuation method changes from *standard price* to either :abbr:`AVCO (Average
Cost)` or :abbr:`FIFO (First In, First Out)` accounting, *revaluation entries* will be automatically
posted to reflect the change in pricing for goods that remain in stock. One entry will be negative
to "remove" the old pricing, and the second entry will be positive to record the new pricing. These
entries are connected to journal entries in Odoo's **Accounting** app.
```

Below is an example of what the :guilabel:`Stock Valuation` table shows when a few stock moves have
occurred for a product using standard price accounting.

.. image:: aging/before-val-method-change.png
   :alt: Stock valuation table in standard price accounting.

Conversely, the following image depicts what the *Stock Valuation* Report table might look like
after a product has switched from standard price to :abbr:`FIFO (First In, First Out)` accounting.

.. image:: aging/after-val-method-change.png
   :alt: Stock valuation table after switching from standard price to FIFO accounting.

.. example::
   The :guilabel:`Remaining Value` and :guilabel:`Remaining Quantity` fields are derived from what
   occurs at the :abbr:`SVL (stock move layer)` level in Odoo and, as such, are better understood
   with an example.

>    `Frankie's Consignment Shop` buys sweaters at the cost, or :guilabel:`Unit Value`, of `5.00`

Reviewer 3: The literal formatting on the company name seemed odd to me, I'd suggest removing the formatting– but totally up to you!
```suggestion
   Frankie's Consignment Shop buys sweaters at the cost, or :guilabel:`Unit Value`, of `5.00`
```

   In the first stock move line item, the :guilabel:`Remaining Quantity` will change from `100.00`
   to `90.00`, once the second stock move is recorded. This change reflects that, although 100
   sweaters were originally purchased, only 90 of those sweaters remain in stock and should be
   counted in the valuation. Similarly, the :guilabel:`Remaining Value` will drop from `$500.00` to
   `$450.00`. The :guilabel:`Total Value` will remain at `$500.00`, regardless of subsequent
   transactions.

   On the other hand, the :guilabel:`Remaining Quantity` of the second stock move line item will be
   recorded and remain at `0.00` because the quantity of `-10.00` was sold. In the system, because
   the :abbr:`SVL (stock move layer)` was a sale, there is no stock left that needs to be valued
   from that transaction.

   .. image:: aging/remaining-val-quant.png
      :alt: Remaining value and quantity are calculated based on :abbr:`SVLs (stock move layers)`.

Change the valuation date
-------------------------

To see the valuation of stock moves at a specific date and time, click the :guilabel:`Valuation At

> Date` button, located in the top-left corner of the Stock Valuation page. The report will show the
> :guilabel:`Quantity` and :guilabel:`Total Value` of each stock move.

Reviewer 3: Add guilabel please :)
```suggestion
Date` button, located in the top-left corner of the :guilabel:`Stock Valuation` page. The report
will show the :guilabel:`Quantity` and :guilabel:`Total Value` of each stock move.
```

.. note::
   The :guilabel:`Remaining Quantity` and :guilabel:`Remaining Value` of the stock moves will *not*
   be point-in-time for any dates chosen in the past. The stock moves shown when selecting a past
   date will still display the *current on-hand quantity and value* of the products.

.. example::
   A business has 100 sofas in stock on January 1st and sells 20 of those sofas on February 1st. The

>    :guilabel:`Remaining Quantity` of the :abbr:`SVL (stock move layer)` will drop from '100.00' to
>    '70.00' on February 1st. If no other stock moves take place, and on February 1st, the :guilabel:
>    `Valuation at Date` is selected as January 1st, the :guilabel:`Remaining Quantity` will still
>    show as '70.00'.

Reviewer 3: adjust number formatting to stay consistent with the other examples in this doc
```suggestion
   :guilabel:`Remaining Quantity` of the :abbr:`SVL (stock move layer)` will drop from `100.00` to
   `70.00` on February 1st. If no other stock moves take place, and on February 1st, the
   :guilabel:`Valuation at Date` is selected as January 1st, the :guilabel:`Remaining Quantity` will
   still show as `70.00`.
```

the :icon:`fa-plus-square` :guilabel:`(plus)` icon in each column or row will reveal a drop-down
list of options to create a more granular breakdown of the inventory valuation. The drop-down
options include: :guilabel:`Product`, :guilabel:`Lot/Serial Number`, :guilabel:`Product Category`,
:guilabel:`Date`, :guilabel:`Company`, or :guilabel:`Add Custom Group` Clicking the
:icon:`fa-minus-square-o` :guilabel:`(minus)` icon collapses the field back to an empty state.

In the table, the :guilabel:`Remaining Qty` column displays the number of on-hand items, and
:guilabel:`Remaining Value` displays the total cost of purchasing these items.

.. image:: aging/stock-aging-report.png
   :alt: Stock aging report, showing product row items and day columns.

.. _inventory/warehouses_storage/graph-view:

Graph view
==========

The stock value can be depicted graphically by clicking the :icon:`fa-area-chart`
:guilabel:`(graph)` icon. By default, the graph is displayed in :icon:`fa-line-chart` line chart
view and filtered to show the cumulative total of all inventory value over time in Odoo.

At the top of the report, a :icon:`fa-bar-chart` bar chart or :icon:`fa-pie-chart` pie chart view

> can be selected instead.

Reviewer 3: I recommend linking to the essentials doc on reporting
```suggestion
can be selected instead.

.. seealso::
   :doc:`Odoo reporting essentials <../../../../essentials/reporting>`
   
```

   from that transaction.

   .. image:: aging/remaining-val-quant.png
      :alt: Remaining value and quantity are calculated based on :abbr:`SVLs (stock move layers)`.

Change the valuation date
-------------------------

To see the valuation of stock moves at a specific date and time, click the :guilabel:`Valuation At
Date` button, located in the top-left corner of the :guilabel:`Stock Valuation` page. The report
will show the :guilabel:`Quantity` and :guilabel:`Total Value` of each stock move.

.. note::
   The :guilabel:`Remaining Quantity` and :guilabel:`Remaining Value` of the stock moves will *not*
   be point-in-time for any dates chosen in the past. The stock moves shown when selecting a past
   date will still display the *current on-hand quantity and value* of the products.

.. example::
   A business has 100 sofas in stock on January 1st and sells 20 of those sofas on February 1st. The
   :guilabel:`Remaining Quantity` of the :abbr:`SVL (stock move layer)` will drop from `100.00` to
   `70.00` on February 1st. If no other stock moves take place, and on February 1st, the
   :guilabel:`Valuation at Date` is selected as January 1st, the :guilabel:`Remaining Quantity`

>    will still show as `70.00`.

Reviewer 4: 100 - 20 = 70 ? :thinking: 

https://x.com/ggellatly/status/1915171240010211416

Reviewer 3: Oops! Addressing here https://github.com/odoo/documentation/pull/13060
Thank you
